packages/shared/src/models/IRecipe.ts

```diff
@@ -57,4 +57,6 @@ export interface Recipe {
    * server the recipe requires
    */
   backend?: string;
+  language?: string;
```

Comment:
A recipe might be backend / frontend so rust in backend and js in frontend, might be a good idea to have it as an array